views/apps/index.html

```diff
@@ -35,7 +35,8 @@ <h3>Categories</h3>
           <li class='listed-app'>
             <a href='/apps/{{this.slug}}' title='{{this.name}} - {{app.description}}'>
               <div class='listed-app-logo-wrapper'>
-                <img class='listed-app-logo' src='/node_modules/electron-apps/apps/{{this.slug}}/{{this.icon64}}' alt='{{app.name}}'>
+                <div class="listed-app-logo-placeholder placeholder" style="background: {{this.goodColorOnWhite}}"></div>
```

Comment:
I love how the placeholder makes use of the good color extracted from the logo! 👍 😍

Reply:
Thanks!